toolkit/pythonit_toolkit/pastaporto/tokens.py

```diff
@@ -24,3 +24,15 @@ def decode_service_to_service_token(
         algorithms=["HS256"],
         options={"require": ["exp", "iss", "aud", "iat"]},
     )
+
+
+def generate_token(secret, issuer: str, audience: str, expires_in: str = "1m"):
+    return jwt.encode(
+        {
+            "issuer": issuer,
+            "audience": audience,
+            "expires_in": expires_in,
```

Comment:
where is this `expires_in` coming from? jwt payload expects `exp` for the expiration

PyJWT docs: https://pyjwt.readthedocs.io/en/latest/usage.html#expiration-time-claim-exp

e.g https://github.com/pythonitalia/pycon/blob/be4ccaeee6253b0e66c1106d5f2e59291696c5fe/users-backend/users/domain/entities.py#L77

Reply:
also it can't be "1m", it has to be a datetime, from the documentation:

```
datetime.datetime.now(tz=timezone.utc) + datetime.timedelta(seconds=30)
```